frontend/src/tests/components/Commons/CommonsPlay.test.js

```diff
@@ -35,4 +36,29 @@ describe("CommonsPlay tests", () => {
             expect(screen.getByTestId("commons-card")).toBeInTheDocument();
         });
     });
+    test("Tests for not logged in", async () => {
```

Comment:
This test can kill one of the mutation tests for `firstname`.

Reply:
Nice